stubs/requests/requests/sessions.pyi

```diff
@@ -8,6 +8,7 @@ from urllib3._collections import RecentlyUsedContainer
 from . import adapters, auth as _auth, compat, cookies, exceptions, hooks, models, status_codes, utils
 from .models import Response
 from .structures import CaseInsensitiveDict as CaseInsensitiveDict
+from .utils import DEFAULT_PORTS as DEFAULT_PORTS
```

Comment:
Any particular reason to re-export this? Looks like at runtime it's a regular import.

Reply:
Yeah, I was on the fence on whether to re-export this or just allowlist it. I'll allowlist it.